libs/auth-api-lib/seeders/20241105120627-add-general-mandate-delegations.js

```
@@ -0,0 +1,61 @@
+'use strict'
```

Biome

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)